### PR TITLE
Exceptions logger

### LINKED ISSUE #5: Log exceptions to file
I have working code locally that supports setting an environment variable to trigger writing exceptions to a file. The issue is, you must set a log level for the transport so it also includes all message using `logger.error()`.

We could switch to syslog levels:

`{ emerg: 0, alert: 1, crit: 2, error: 3, warning: 4, notice: 5, info: 6, debug: 7 }`

This would allow setting the exception logger to only include `emerg` messages, which we just won't use. We would need to use `logger.warning()` in place of `logger.warn()`.

### PR DESCRIPTION
**Pull request by jackboberg**
This is a work in progress, only adding as a PR to make it easier to give feedback.
- [x] Get feedback from team on this approach
- [x] Add tests
- [x] Update Readme

Closes #5

### CODE REVIEW

**Comment by jackboberg** [comment]
Moving to a separate module, per discussion IRL.

**Comment by jackboberg** [comment]
Re-opening to explore another idea of keeping this together. I have updated code, finalizing tests and will push probably tomorrow.

**Comment by jackboberg** [comment]
Refactor complete, please reconsider this PR.

File: README.md [hunk 1]
```diff
@@ -35,7 +35,38 @@ variable `LOG_LEVEL`.
...
+
+Logger.writeExceptions(PATH);
+
+// you can still use the factory to get a logger instance for this file
+var logger = Logger('namespace');
+logger.info('message', { example: true });
+```
+
+Note, you must use the required module directly, and create a logger instance
+separately. You typically only need to do this in your main file, so the
```
Comment (franvarney) [inline README.md:52]: Do you mean that you typically only need to create the logger instance separately in the main file?

So in the for example, in `index.js` you might have:

```
const Logger = require('@modulus/logger');
Logger.writeExceptions(PATH);

var logger = Logger('namespace');
logger.info('message');
```

while in another file you might have:

```
const Logger = require('@modulus/logger')('namespace');
Logger.info('message');
```

Comment (jackboberg) [inline README.md:52]: exactly

**Comment by franvarney** [comment]
:shipit:

**Comment by tzmanics** [comment]
:+1:

**Pull request merged**